include/functions.h

```diff
@@ -701,10 +701,10 @@ Vec3s* Camera_GetCamDir(Vec3s* dir, Camera* camera);
 s16 Camera_GetCamDirPitch(Camera* camera);
 s16 Camera_GetCamDirYaw(Camera* camera);
 s32 Camera_AddQuake(Camera* camera, s32 arg1, s16 y, s32 countdown);
-s32 Camera_SetParam(Camera* camera, s32 param, void* value);
-s32 func_8005AC48(Camera* camera, s16 arg1);
-s16 func_8005ACFC(Camera* camera, s16 arg1);
-s16 func_8005AD1C(Camera* camera, s16 arg1);
+s32 Camera_SetViewParam(Camera* camera, s32 viewFlag, void* param);
+s32 Camera_OverwriteStateFlags(Camera* camera, s16 stateFlags);
+s16 Camera_SetStateFlags(Camera* camera, s16 stateFlags);
+s16 Camera_UnsetStateFlags(Camera* camera, s16 stateFlags);
```

Comment:
Thoughts on:
```suggestion
s32 Camera_OverwriteStateFlags(Camera* camera, s16 stateFlags);
s16 Camera_SetStateFlagsMask(Camera* camera, s16 stateFlags);
s16 Camera_ClearStateFlagsMask(Camera* camera, s16 stateFlags);
```
`Camera_OverwriteStateFlags` is fine to me, or could also be (but I don't see these as particularly better):
- `Camera_SetStateFlagsWhole`
- `Camera_ReplaceStateFlags`

Reply:
I'm not sure what `Mask` adds to these names. You're just setting flags right? So like:
`Camera_ClearStateFlags(camera, CAM_STATE_UNDERWATER);`
You're setting a flag, not a mask.

I would say both `Camera_OverwriteStateFlags` and `Camera_ReplaceStateFlags` seems fine to me.
Maybe even `Camera_ClearAndStateFlags` just to emphasize that any flag is cleared that isn't set in the arg.

Reply:
Well `stateFlags` is a variable so `SetStateFlags` sounds like it would do `stateFlags = `, not `stateFlags |=`

with "Mask" I would understand "set state flags bits from this mask `stateFlags |= mask`"

Reply:
It's the bitwise nature of it implied by `flags`? So setting a flag means `|=`, not `=`, at least that seems more intuitive to me. Setting a mask makes me think you're blocking flags from being set by setting a mask or something